### PR TITLE
Feature: manage content object states

### PR DESCRIPTION
**Pull request by benkmugo**
- added state assign, view functions
- updated documentation

### CODE REVIEW

**Comment by benkmugo** [comment]
Addresses #36

File: docs/modules_contentobject.md [hunk 1]
```diff
@@ -148,3 +151,21 @@ Removes a translation from the content object
...
 ```
 
+## stateassignbyid
+Assigns an object state by state id
+```sh
+eep contentobject stateassignbyid <object id> <state id>
+```
+
+## stateassignbyidentifier
+Assigns an object state by state/group identifier e.g. ez_lock/locked
```
Comment (dougplant) [inline docs/modules_contentobject.md]: Could you make at a suggestion for at least one location where a person would find some of these identifiers, like an INI or code-file? Doesn't have to be an anything more than "See, for example, settings/states.ini" or similar.

**Comment by dougplant** [comment]
just one suggestion to add to the help text

File: docs/modules_contentobject.md (hunk 1 above)
Comment (benkmugo) [reply docs/modules_contentobject.md]: Good idea. Doc update has been added.

**Pull request merged**